src/App.tsx

```diff
@@ -1,61 +1,121 @@
+import { useState } from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo, User, TodosWithUser } from './types/Types';
 
 export const App = () => {
+  const users: User[] = usersFromServer;
+  const todos: Todo[] = todosFromServer;
+
+  function preparedTodosWithUsers(): TodosWithUser[] {
+    return (
+      todos.map(todo => ({
+        ...todo,
+        user: users.find(user => user.id === todo.userId) || null,
+      }))
+    );
+  }
```

Comment:
why do you need return here?
```suggestion
  function preparedTodosWithUsers(): TodosWithUser[] (
      todos.map(todo => ({
        ...todo,
        user: users.find(user => user.id === todo.userId) || null,
      )))
    );
  }
```